sync/class.jetpack-sync-sender.php

```diff
@@ -54,12 +54,16 @@ private function init() {
 
 	public function maybe_set_user_from_token() {
 		$jetpack = Jetpack::init();
-		if ( $jetpack->verify_xml_rpc_signature() ) {
+		if ( defined( 'XMLRPC_REQUEST' ) &&
```

Comment:
I think we could use the `Jetpack_Constants::is_true( 'XMLRPC_REQUEST' )` instead so that we can write tests for that easier?